ansible/roles/test/files/ptftests/arista.py

```diff
@@ -313,6 +331,49 @@ def parse_bgp_route(self, output, expects):
 
         return set(expects) == prefixes
 
+    def get_bgp_neighbors(self):
```

Comment:
>get_bgp_neighbors [](start = 8, length = 17)

You return both neigh and dut, why naming function with only neighbors?

Reply:
Planning to rename this to get_bgp_info()